### PR TITLE
Fix bug squid installation on stratum1

### PR DESCRIPTION
**Pull request by ahmam**
Add variable to make choose  to configure squid in stratum 1 or not.

### CODE REVIEW

**Comment by btravouillon** [comment]
Has this bug been reported to upstream project and acknowledged?

**Review by milaflq — COMMENTED** [review]
@ahmam
[Add variable cvmfs_strqtum1_squid](https://github.com/mila-iqia/ansible-cvmfs/pull/2/commits/54b87be7c0769ab015c2ab700cea53f13afaed2b)

Est-ce un typo? (cvmfs_strqtum1_squid)

File: defaults/main.yml [hunk 1]
```diff
@@ -20,6 +20,8 @@ cvmfs_localproxy_http_ports:
 
 cvmfs_stratum1_apache_port: 8008
 cvmfs_stratum1_cache_mem: 128 # MB
+# Automatically configure a squid on startum1.
```
Comment (milaflq) [inline defaults/main.yml]: "startum1"
Typo?

**Review by milaflq — APPROVED** [review]
Sinon, le code est bon de mon point de vue.

File: defaults/main.yml (hunk 1 above)
Comment (ahmam) [reply defaults/main.yml]: oui c'est type je corrige

**Comment by ahmam** [comment]
> Has this bug been reported to upstream project and acknowledged?

it is not reported to upstream because it is really optional it is a feature that we need but I will report it in a pull request.

**Comment by btravouillon** [comment]
> it is not reported to upstream because it is really optional it is a feature that we need but I will report it in a pull request.

Looks like package `squid` is also added in upstream PR that you mentionned in #3. Looking further, the `squid` package is installed on the stratum1 without condition for RHEL-like hosts (https://github.com/galaxyproject/ansible-cvmfs/blob/0.2.21/vars/redhat.yml#L19). Thus I believe you can just add the package, you don't have to introduce any new variable that would change the behaviour of the role for RHEL-like hosts.

**Comment by ahmam** [comment]
> > it is not reported to upstream because it is really optional it is a feature that we need but I will report it in a pull request.
> 
> Looks like package `squid` is also added in upstream PR that you mentionned in #3. Looking further, the `squid` package is installed on the stratum1 without condition for RHEL-like hosts (https://github.com/galaxyproject/ansible-cvmfs/blob/0.2.21/vars/redhat.yml#L19). Thus I believe you can just add the package, you don't have to introduce any new variable that would change the behaviour of the role for RHEL-like hosts.

I just added the package  squid to comply with redhat and remove the changes made by the variable.

**Review by btravouillon — APPROVED** [review]
Looks good to me. 👍

**Pull request merged**